pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -364,6 +364,11 @@ func classifyVolumesForMigration(vmi *v1.VirtualMachineInstance) *migrationDisks
 			(volSrc.HostDisk != nil && *volSrc.HostDisk.Shared) {
 			disks.shared[volume.Name] = true
 		}
+
```

Comment:
the idea was to explain that even if a volume is, for example, a ConfigMap it does not necessarily imply that it will have an .iso generated (something that is assumed in other parts of the code).

I think it would be best to remove the comment and verify that the volume has a Disk associated with it.